src/main/java/net/snowflake/client/core/bind/BindUploader.java

```diff
@@ -64,6 +68,9 @@
   // size (bytes) per file in upload, 100MB default
   private long fileSize = 100 * 1024 * 1024;
 
+  // size (bytes) of max input stream (10MB default)
+  private long inputStreamBufferSize = 1024 * 1024 * 10;
```

Comment:
should be a constant right?

Reply:
This can now be changed by the setInputStreamBufferSize() function (for testing purposes).